code/modules/mob/living/simple_animal/borer.dm

```diff
@@ -1040,7 +1040,7 @@ var/global/borer_unlock_types_leg = typesof(/datum/unlockable/borer/leg) - /datu
 			//testing("Client of [G] inexistent")
 			continue
 
-		//#warning Uncomment me.
+		//#warn Uncomment me.
```

Comment:
lmao

Reply:
I honestly cannot comprehend what the fuck this line hoped to achieve